models/User.ts

```diff
@@ -1,15 +1,49 @@
-import type { roles, province, locale } from "@prisma/client";
+import {
+    roles,
+    province,
+    locale,
+    difficulties,
+    therapy,
+    heardFrom,
+} from "@prisma/client";
+
+export { roles, province, locale, difficulties, therapy, heardFrom };
 
 /* Input type for Parent inputs */
 export type ParentInput = {
+    // Parent Information
+    // Note: first/last name are specified in UserInput
     phoneNumber: string;
     isLowIncome?: boolean;
+    preferredLanguage: locale;
+    proofOfIncomeLink?: string;
+    heardFrom?: heardFrom[];
+    heardFromOther?: string;
+
+    // Child Information
+    childFirstName: string;
+    childLastName: string;
+    childDateOfBirth: Date;
```

Comment:
it seems like the fields here are similar to the ones in `CreateStudentInput` - can we not just nest that type in here?

Reply:
I made the change [here](https://github.com/uwblueprint/social-diversity-for-children/pull/65/commits/dc7251ec6ca5096375c4080686ec84f0f8fc15e3). @mahadzaryab1 @jason17h @sohak22, could you guys take a look?